client/src/components/SignUp.js

```diff
@@ -132,17 +140,25 @@ function SignUp(props) {
         <form>
           <h2 className={classes.h2}>Sign up</h2>
           <label>Your Name</label>
-          <TextField
-            className={classes.textField}
-            variant="outlined"
-            label="name"
-            fullWidth
-            required
-            type="text"
-            error={!!errors.name}
-            helperText={errors.name}
-            onChange={(e) => setName(e.target.value)}
-          />
+          <Tooltip
+            open={openNameTooltip}
+            title={errors.name}
+            arrow
+            disableHoverListener
+            disableTouchListener
```

Comment:
I used required attribute to make sure the inputs are filled by the user

Reply:
Please check the last commit on this PR to see my changes

Reply:
Looks good, thank you!